force-app/main/default/lwc/crmThemeCategorization/crmThemeCategorization.js

```diff
@@ -340,6 +365,15 @@ export default class CRMThemeCategorization extends LightningElement {
         return disabled;
     }
 
+    get themeGruopVisible() {
+        return this.variant === 'HIDE_THEME_GROUP' || this.variant === 'HIDE_THEME_GROUP_AND_SUBTHEME' ? false : true; 
+    }
+
+    get subthemeVisible() {
+        return this.variant === 'HIDE_SUBTHEME' || this.variant === 'HIDE_THEME_GROUP_AND_SUBTHEME' ? false : true;  
+    }
```

Comment:
```suggestion
    get themeGruopVisible() {
        return !(this.variant === 'HIDE_THEME_GROUP' || this.variant === 'HIDE_THEME_GROUP_AND_SUBTHEME'); 
    }

    get subthemeVisible() {
        return !(this.variant === 'HIDE_SUBTHEME' || this.variant === 'HIDE_THEME_GROUP_AND_SUBTHEME');  
    }
```
Trur dette passer bedre

Reply:
👍